web/src/components/change.js

```diff
@@ -30,6 +30,8 @@ class ChangeStatus extends React.Component {
         return <Badge variant="success">Open</Badge>
       case 'MERGED':
         return <Badge variant="primary">Merged</Badge>
+      case 'CLOSED':
+        return <Badge variant="danger">Closed</Badge>
```

Comment:
Why not using the Abandoned term ?

Reply:
do we want to change CLOSED to ABANDONED too?

Reply:
No, let's keep like that. It is called at various places closed in the backed.